src/sass/pages/index.scss

```diff
@@ -196,7 +197,7 @@
   }
 
   &__content {
-    @include columns(8);
+    @include columns(10);
```

Comment:
Yes, but this does not apply the redesign, just makes everything look ok with the new grid.